src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/JsonLogin/config.yml

```diff
@@ -16,7 +16,7 @@ security:
             pattern: ^/
             anonymous: true
             json_login:
-               check_path:    /mychk
+               check_path:    /chk
```

Comment:
Indeed :blush: I think initially it was not stateless and used the check_path:

https://github.com/symfony/symfony/pull/18952#issuecomment-224635760